mbofui/app/js/utils.js

```diff
@@ -60,6 +60,21 @@ var leafletize = function(data, user){
     return leafletList;
 };
 
+
+var normalizeaddressLookupResults = function(data){
+  var locationList = [];
+  _.each(data, function(location){
+    locationList.push({
+      locationName: location.display_name || location.formatted_address,
+      lat:location.lat || location.geometry.location.lat,
+      lng:location.lon || location.geometry.location.lng
```

Comment:
So the data comes from OpenStreet or Google. They have different formats and key names.

Here we are basically saying  - "choose from either (both cannot be true at the same time)"

Yeah - it would be nice if OpenStreet and Google called them the same thing. When we convert the data for internal use we are going with lat and lng, so we are internally consistent.